README.md

```diff
@@ -28,7 +28,8 @@ $ go get github.com/hacksoc-manchester/www
 Make sure [Git is installed](http://git-scm.com/downloads) on your machine and referenced by your system's ```PATH```.
 
 #### Environment Variables
-Linux/MacOS - Add the following to the `.env` file in your project root (create it if it does not exist, but do *not* commit it)
+Create a file called `.env` in the project root directory, and make sure it is added in `.gitignore`. (On Windows Explorer you may have to name the file `.env.`)
```

Comment:
I'd rewrite this as:

> Create a file called `.env` in the project root directory and make sure it's added in `.gitignore` (if using Windows, you might have to name it `.env.`). Paste the following text in the file and edit the variable values sensibly:

Although are we sure that the _"if using Windows, you might have to name the file `.env.`"_ part is correct?

Reply:
>Although are we sure that the "if using Windows, you might have to name the file .env." part is correct?

Yes, I have also tested it myself.